app/src/main/java/com/metrostateics499/vre_app/KeyPhrase.kt

```diff
@@ -7,7 +7,7 @@ package com.metrostateics499.vre_app
  * @property keyPhrase
  * @constructor Create empty Key phrase
  */
-class KeyPhrase(var keyPhrase: String) {
+data class KeyPhrase(var keyPhrase: String) {
     override fun toString(): String {
         return keyPhrase
     }
```

Comment:
I think we might not need this because the implicit get() method that we get for free with Kotlin data classes should already return the keyPhrase object. 
I'm thinking instead of doing KeyPhrase.toString() you would do KeyPhrase.keyPhrase which would call KeyPhrase.keyPhrase.get()
Test this out first though to confirm.